autopilot/contractor/contractor.go

```diff
@@ -1205,15 +1205,15 @@ func performContractFormations(ctx *mCtx, bus Bus, w Worker, cr contractReviser,
 		// prepare a gouging checker
 		logger := logger.With("hostKey", candidate.host.PublicKey).
 			With("remainingBudget", remainingFunds).
-			With("subnets", candidate.host.Subnets)
+			With("addresses", candidate.host.ResolvedAddresses)
```

Comment:
`// prepare a gouging checker` is copied over I think and should be `// prepare a logger` (?)